src/components/TrustedTimestamping/AllFaqs.vue

```diff
@@ -1,7 +1,8 @@
 <template lang="pug">
 div
-  div(v-for="faq in faqs")
+  div(v-for="(faq, index) in faqs")
     faq.mb-3(
+      :index="index"
```

Comment:
Depending on what we decide about the FAQ component and passing index, I don't think we will need this part or to pass in index.